tests/bluetooth/common/testlib/include/bs_helpers/testcase.h

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "bs_tracing.h"
+#include "bs_types.h"
+#include "bstests.h"
+
+extern enum bst_result_t bst_result;
+
+#define TEST_START(timeout_us, message)                                                            \
```

Comment:
Please provide a description of the macro purpose and parameters (also for FAIL)